stock_cycle_count/models/stock_location.py

```diff
@@ -21,13 +21,12 @@
 class StockLocation(models.Model):
     _inherit = "stock.location"
 
-    @api.multi
     def _compute_loc_accuracy(self):
         for rec in self:
             history = self.env["stock.inventory"].search(
-                [("location_id", "=", rec.id), ("state", "=", "done")]
+                [("location_ids", "in", rec.id), ("state", "=", "done")],
+                order="write_date desc",
```

Comment:
Why adding this ?

Reply:
I think it was added by mistake when changing another search on "stock.inventory", I will correct it.